README.md

```diff
@@ -682,6 +682,22 @@ Supported options:
 --formatter-junit-dump-xml=     path to a dumped xml file [default: "./junit-report.xml"]
 ```
 
+### GithubActions Formatter
```

Comment:
Can you also add a link in `Table of Contents`-list ? `1. [GitHubActions Formatter](#githubactions-formatter)` should be correct.

Ahh and it should be `GitHubActions` ;-)

Reply:
> Can you also add a link in Table of Contents-list ? 1. [GitHubActions Formatter](#githubactions-formatter) should be correct.

Sure.

> Ahh and it should be GitHubActions ;-)

Why GitHubActions? The Formatter class is `GithubActionsOutputFormatter` with a lower case `h` in Github.